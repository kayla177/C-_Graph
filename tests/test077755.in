ENTITY 1UQIIZ0N name_1UQI paper
ENTITY 4J2KVU13 name_4J2K institution
ENTITY ZI23RP0U name_ZI23 journal
ENTITY HEYIHK4U name_HEYI author
ENTITY H1BLW8NZ name_H1BL paper
ENTITY 40ZA4T0X name_40ZA paper
ENTITY ES69CZQK name_ES69 conference
ENTITY UX7NK9CV name_UX7N journal
ENTITY 2IJOI8YI name_2IJO conference
ENTITY QKMP7TOY name_QKMP paper
RELATIONSHIP ZI23RP0U collaborated_with 2IJOI8YI 3.2
RELATIONSHIP ZI23RP0U cited_by 1UQIIZ0N 7.9
RELATIONSHIP 2IJOI8YI authored_by 1UQIIZ0N 2.1
RELATIONSHIP 1UQIIZ0N published_in H1BLW8NZ 1.1
RELATIONSHIP 4J2KVU13 cited_by 1UQIIZ0N 10.0
RELATIONSHIP 1UQIIZ0N collaborated_with ES69CZQK 5.5
RELATIONSHIP 4J2KVU13 affiliated_with 2IJOI8YI 5.2
RELATIONSHIP QKMP7TOY affiliated_with 40ZA4T0X 2.8
RELATIONSHIP ES69CZQK presented_at HEYIHK4U 8.2
RELATIONSHIP ZI23RP0U authored_by UX7NK9CV 6.0
RELATIONSHIP 4J2KVU13 published_in 40ZA4T0X 6.7
RELATIONSHIP 4J2KVU13 presented_at ES69CZQK 1.3
RELATIONSHIP QKMP7TOY presented_at 1UQIIZ0N 9.7
RELATIONSHIP QKMP7TOY authored_by 4J2KVU13 2.1
RELATIONSHIP ZI23RP0U affiliated_with QKMP7TOY 8.6
DELETE H1BLW8NZ
DELETE HEYIHK4U
PRINT ZI23RP0U
DELETE QKMP7TOY
DELETE 1UQIIZ0N
DELETE 2IJOI8YI
PRINT ZI23RP0U
PRINT UX7NK9CV
PRINT UX7NK9CV
EXIT
