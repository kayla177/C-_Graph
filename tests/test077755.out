success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
2IJOI8YI 1UQIIZ0N UX7NK9CV QKMP7TOY
success
success
success
UX7NK9CV
ZI23RP0U
ZI23RP0U
